betterocr/detect.py

```diff
@@ -70,9 +70,14 @@ def detect_text(
     print("=====")
     print(prompt)
 
-    api_key = os.environ["OPENAI_API_KEY"]
-    if "API_KEY" in options["openai"] and options["openai"]["API_KEY"] != "":
-        api_key = options["openai"]["API_KEY"]
+    # Prioritize user-specified API_KEY
+    api_key = options["openai"].get("API_KEY", os.environ.get("OPENAI_API_KEY"))
```

Comment:
wtf i was too javascript these days and forgot about `dictionary.get` lol thx 👍

Reply:
I do the same thing, very often lol